coderd/workspacebuilds_test.go

```diff
@@ -229,7 +229,7 @@ func TestPatchCancelWorkspaceBuild(t *testing.T) {
 		var err error
 		build, err = client.WorkspaceBuild(context.Background(), build.ID)
 		require.NoError(t, err)
-		return build.Job.Status == codersdk.ProvisionerJobCanceled
+		return build.Job.Status == codersdk.ProvisionerJobFailed // this is timing out
```

Comment:
Ditto

Reply:
Probably the same reason as above! I'll take a look in https://github.com/coder/coder/issues/3083